vendor/bat-native-ads/src/bat/ads/internal/container_util.h

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_ADS_INTERNAL_CONTAINER_UTIL_H_
+#define BAT_ADS_INTERNAL_CONTAINER_UTIL_H_
+
+#include <iterator>
+#include <deque>
+#include <vector>
+
+namespace ads {
+
+template<typename T>
+std::vector<std::vector<T>> SplitVector(
+    const std::vector<T>& elements,
+    const int chunk_size) {
+  std::vector<std::vector<T>> result;
```

Comment:
`DCHECK` that `chunk_size != 0` (since we're going to use it as a divisor)?

Reply:
Fixed